addons/test_website/models/website.py

```diff
@@ -1,14 +1,7 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import models
+from odoo.addons.website.models.website import SEARCH_TYPE_MODELS
+from odoo.tools import OrderedSet
 
-
-class Website(models.Model):
-    _inherit = "website"
-
-    def _search_get_details(self, search_type, order, options):
-        result = super()._search_get_details(search_type, order, options)
-        if search_type in ['test']:
-            result.append(self.env['test.model']._search_get_detail(self, order, options))
-        return result
+SEARCH_TYPE_MODELS['test'] |= OrderedSet(('test.model',))
```

Comment:
Why not simply:
```py
SEARCH_TYPE_MODELS['test'] |= ['test.model']
```
Arrays are also ordered iterables.

Reply:
The operator doesn't work with a regular `set`, but does with the `OrderedSet`, thanks!